docs/staticAnalysis.md

```diff
@@ -25,14 +25,23 @@ Conversely, when updating any tool, ensure that the tool version is supported by
 [CheckStyle](http://checkstyle.sourceforge.net/index.html) helps to enforce coding standard in Java source code.
 The rules to be used are configured in a ruleset file; in TEAMMATES the file can be found [here](../static-analysis/teammates-checkstyle.xml).
 The plugin for Eclipse can be found [here](http://eclipse-cs.sourceforge.net/#!/).
+The plugin for IntelliJ can be found [here](https://plugins.jetbrains.com/idea/plugin/1065-checkstyle-idea).
```

Comment:
> It's better if these two links are moved to the respective sections below. For example, the first point in the instructions can be,

Should the Eclipse one be moved as well? This was originally with the IntelliJ section but wilson requested for it to be moved up.